### ISSUE TITLE
make: *** No targets specified and no makefile found.  Stop.

### ISSUE THREAD

**Issue by programmerShinobi** [EnterpriseDB/mysql_fdw#305]
faqih@shinobi:/usr/local/pgsql/bin$ sudo make USE_PGXS=1
make: *** No targets specified and no makefile found.  Stop.

**Comment by devrimgunduz** [comment]
@programmerShinobi really? This is how you report a bug?

**Comment by programmerShinobi** [comment]
> @programmerShinobi really? This is how you report a bug?

yes really, and then.. now i congratulation using tutorial this link : https://gist.github.com/programmerShinobi/6a09c0cc7f3089df3d7305d93f258381

**Comment by surajkharage19** [comment]
Hi @programmerShinobi,

Just checking if you need further assistance on this or we are good to close this issue?
